src/plugins/graph/components/attribute-label.tsx

```diff
@@ -162,7 +161,7 @@ export const AttributeLabel = observer(
     }, [place, dataConfiguration, refreshAxisTitle]);
 
     const readyForPortal = parentElt && onChangeAttribute && onTreatAttributeAs && onRemoveAttribute;
-    const skipPortal = appConfig.getSetting("defaultSeriesLegend", "graph") && place === "left";
+    const skipPortal = useSettingFromStores("defaultSeriesLegend", "graph") && place === "left";
```

Comment:
@kswenson ah ok, thanks, there are a few other spots I can make this same change.